plugins/modules/system/sap_snote.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com> <rainer.leber@sva.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sap_snote
+
+short_description: This module will upload and (de)implements C(SNOTES) in a SAP S4HANA environment.
+
+version_added: "1.0.0"
+
```

Comment:
just to be safe "0.2.0"?

Reply:
Next version would be 1.0.0 Because of package integration

Reply:
👍